colossalai/zero/low_level/low_level_optim.py

```diff
@@ -238,7 +238,7 @@ def _create_master_param_current_rank(self, param_list):
         device = "cpu" if self._cpu_offload else get_accelerator().get_current_device()
 
         for param in param_list:
-            padding_size = (self._world_size - param.numel() % self._world_size) % self._world_size
+            padding_size = self._world_size - param.numel() % self._world_size
```

Comment:
This is wrong. What if `param.numel() % self._world_size == 0`? The previous is right.